### ISSUE TITLE
Accessibility Bug: Accessibility issue on selecting "Browse QA Challenges" in QA Competition Types Page #280

### ISSUE THREAD

**Issue by suganyapravin** [topcoder-platform/Training-Challenges#280]
Steps to reproduce:

Go to https://www.topcoder.com/
Select Explore-->Thrive-->QA-->Help Center-->QA Competition Types
Goto the bottom of the page and click on "Browse QA Challenges"

Actual Result:
Blank White screen is displayed if there is no challenges to show

Expected Result:
Display "No Challenges to show "message and stay in the same page
![Screen Shot 2021-10-17 at 11 19 10 PM](https://user-images.githubusercontent.com/59360647/137665262-73f81604-07b6-46f0-8063-8f33ac9b359f.png)
.
![Screen Shot 2021-10-17 at 10 57 40 PM](https://user-images.githubusercontent.com/59360647/137665293-c9d34c36-3e9d-47bd-82b3-0bf23d1296b3.png)
![Screen Shot 2021-10-17 at 10 57 26 PM](https://user-images.githubusercontent.com/59360647/137665319-3007bf2c-da2b-47d8-b2dd-dd34c6046e8e.png)

**Comment by sdgun** [comment]
Good catch @suganyapravin In this type of issues it will be helpful if you can take the screenshot with the console errors(in dev tools)
